frontend/src/framework/internal/EnsembleSetLoader.ts

```diff
@@ -357,20 +349,6 @@ async function loadEnsembleApiDataMapFromBackend(
             continue;
         }
```

Comment:
I know we did not check it previously, because the back-end failed if there was no parameters. But should we have a check for `parameterArray.length === 0` to ensure that the ensemble has non-empty parameter array?